source/main.c

```diff
@@ -111,6 +111,9 @@ static void stop_install_check(void)
     }
     for(int i = 0; i < MODE_AMOUNT; i++)
     {
+        if(installCheckThreads[i] == NULL)
```

Comment:
style: snake_case naming; I know this was already here but change it if it's not too much work